HopBot/MapService/FileService.cs

```diff
@@ -286,7 +286,39 @@ public void ExtractFile(string compressedFile, out string fileName, bool acerMet
                 _log.LogError(e.Message);
             }
         }
+        
+	//New Function :v
+	    public void CopyToFastdl(string filename)
```

Comment:
I don't really get the usage of this method could you please explain it for what it would be used. The "CompressToFastdl" method already does the work and I don't think .bsp files should be in the FastDl folder. I don't know how the FastDl folder in CS:GO is working but I think only bz2 files should be there, an explanation would be cool.

Reply:
Sorry, not needed for CSS, just CS:GO due to limitations with unzipping maps > 150mb.